integration/concurrency_sqlite/concurrency_sqlite_suite_test.go

```diff
@@ -1,13 +0,0 @@
-package concurrency_sqlite_test
```

Comment:
This part of test failed sometimes (frequent but not always) in pipeline, it might be due to performance of concourse VM, there used goroutine running in multiple Garden containers in one VM. I think there should bet a better way to control goroutine, but I failed, it might need your help to take a look when you travel to Beijing.